cli/src/main/java/io/kestra/cli/App.java

```diff
@@ -77,33 +103,41 @@ protected static void execute(Class<?> cls, String[] environments, String... arg
         applicationContext.close();
 
         // exit code
-        System.exit(Objects.requireNonNullElse(exitCode, 0));
+        return Objects.requireNonNullElse(exitCode, 0);
```

Comment:
```suggestion
        return exitCode, 0;
```

Exit code cannot be null here as it's set in all code path and, well, it's an int which cannot be null